sky/clouds/aws.py

```diff
@@ -317,7 +317,7 @@ def make_deploy_resources_variables(
             'region': region_name,
             'zones': ','.join(zone_names),
             'image_id': image_id,
-            **AWS._get_disk_specs(r.disk_tier or 'low')
+            **AWS._get_disk_specs(r.disk_tier or 'medium')
```

Comment:
It might be better to have a constant called `_DEFAULT_DISK_TIER` in the `Cloud` class, and let the inherited classes use that.
Also, it might be better to handle the None->'_DEFAULT_DISK_TIER` inside the method `_get_disk_specs` to reduce the abstraction leakage. : )